lib/cli/src/Cardano/CLI.hs

```diff
@@ -1538,13 +1556,38 @@ initTracer
 initTracer loggerName outputs = do
     cfg <- do
         c <- defaultConfigStdout
-        CM.setSetupBackends c [CM.KatipBK, CM.AggregationBK]
+        CM.setSetupBackends c [CM.KatipBK, CM.AggregationBK, CM.EKGViewBK, CM.EditorBK]
+        CM.setDefaultBackends c [CM.KatipBK, CM.EKGViewBK]
         CM.setSetupScribes c $ map mkScribe outputs
         CM.setDefaultScribes c $ map mkScribeId outputs
+        CM.setEKGBindAddr c $ Just (Endpoint ("127.0.0.1", 13788))
+        CM.setPrometheusBindAddr c $ Just ("127.0.0.1", 13798)
+        CM.setBackends c "cardano-wallet.metrics" (Just [CM.EKGViewBK])
         pure c
     (tr, sb) <- setupTrace_ cfg loggerName
+    EKG.plugin cfg tr sb >>= loadPlugin sb
+    startCapturingMetrics tr
     pure (sb, (cfg, tr))
 
+  where
+    startCapturingMetrics :: Trace IO Text -> IO ()
```

Comment:
This is a nice copy paste of cardano-node.

Reply:
A link to the original in a comment wouldn't hurt, such that we can compare them and see if the original change in the future, etc